Makefile

```diff
@@ -221,28 +221,21 @@ push-cache:
 # Staging
 # TODO: PC->SAN: see ops/travis/system-testing/build_and_run. Could move images FREFIX and TAG there
 
-.PHONY: build-staging push-staging pull-staging create-staging-stack-file
-# target: build-staging – Builds service images and tags them as 'staging'
-build-staging:
-	export DOCKER_IMAGE_PREFIX=itisfoundation/; \
-	export DOCKER_IMAGE_TAG=staging-latest; \
-	${MAKE} build
+.PHONY: tag push pull create-staging-stack-file
+#target: tag – Tags service images
+tag:
+	for i in $(SERVICES_LIST); do \
+		${DOCKER} tag services_$$i:latest ${DOCKER_IMAGE_PREFIX}$$i:${DOCKER_IMAGE_TAG}; \
```

Comment:
We should account for the variable ``DOCKER_IMAGE_PREFIX`` instead of hard-code ``service_`` suffix
I actually would like the default to be ``DOCKER_IMAGE_PREFIX=simcore_`` someday :-)

Reply:
it's coming with the CI/CD improvements I hope